spec/support/helpers.rb

```diff
@@ -191,6 +191,8 @@ def url_for_resource_collection(resource)
       url_for_resource(resource.provider) + "/courses"
     elsif resource.is_a? AccessRequest
       "/access_requests"
+    elsif resource.is_a? Organisation
+      "/organisations"
```

Comment:
We should raise an exception here to make it more obvious where we've missed adding something to this rickety framework.

Reply:
Done